va/va.h

```diff
@@ -381,6 +381,25 @@ typedef enum
      * (VAEncFEIMVBufferType, VAEncFEIMBModeBufferType) are needed for PAK input.
      **/
     VAEntrypointFEI         = 11,
+    /**
+     * \brief VAEntrypointStats
+     *
+     * Statistics for FEI and non-FEI. A pre-processing function for getting some
```

Comment:
What does it mean my "Statistics for FEI and non-FEI" ? Ideally, VAEntrypointStats is a generic entrypoint for all codecs, If so why should we mention FEI, non-FEI terms here?

Reply:
it only means VAEntrypointStats does not limit to only support FEI use case,  do you think it better to not mention FEI here?